www/src/states/use-chat-state.ts

```diff
@@ -85,6 +85,9 @@ const useChatStoreBase = create<ChatStore>()(
       removeChat: (id) =>
         set((state) => {
           delete state.chats[id];
+          if (state.selectedChatId === id) {
+            state.selectedChatId = Object.keys(state.chats)[0] || null;
+          }
```

Comment:
🤔 this will be null if there had been only 1 chat when you called removeChat, right? is that what we expect?